src/sparsezoo/v2/inference/inference_runner.py

```diff
@@ -112,14 +113,12 @@ def validate_with_onnx_runtime(self) -> bool:
         validation = []
 
         sample_outputs = self._check_and_fetch_outputs(engine_name="onnxruntime")
-
         for target_output, output in zip(sample_outputs, self._run_with_onnx_runtime()):
             target_output = list(target_output.values())
-            is_valid = [
-                numpy.allclose(o1, o2.flatten(), atol=1e-5)
-                for o1, o2 in zip(target_output, output)
-            ]
-            validation += is_valid
+            for o1, o2 in zip(target_output, output):
+                if o2.ndim != o1.ndim:
+                    o2 = o2.squeeze(0)
```

Comment:
Is this to account for batch dimensions? Will there ever be an `o2` that has fewer than 1 dimension? ie. `o2.ndim > o1.ndim + 1` in which case the squeeze will still not make the ndims match

Reply:
Yes, this is to account for batch dimensions. Well, I am 99% sure that `o2.ndim` will always be `o1.ndim + 1` (this seems to be the case for all our integrations). Actually, I should probably remove the `if` statement here and apply the `squeeze` directly.